plugins/com.google.cloud.tools.eclipse.util/templates/appengine/web.xml.ftl

```diff
@@ -1,9 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
+<#if version == "2.5">
 <web-app xmlns="http://java.sun.com/xml/ns/javaee"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xmlns:web="http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd"
          xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd" 
          version="2.5">
+<#elseif version == "3.1">
```

Comment:
This should either be two templates or we should parameterize the attribute values rather than the entire template.